src/lib/playerDat.js

```diff
@@ -34,7 +34,10 @@ async function read (uuid, spawnPoint, worldFolder) {
         pitch: playerData.Rotation.value.value[1],
         onGround: Boolean(playerData.OnGround.value)
       },
-      inventory: playerData.Inventory.value.value
+      inventory: playerData.Inventory.value.value.map(nbtItem => {
+        if (nbtItem.tag) nbtItem.tag.name = ''
```

Comment:
Why

Reply:
nbtItem.tag.name would be undefined otherwise and would cause an error on join.

Reply:
if it's defined and not empty you're losing the value
but it sounds like the problem should be fixed wherever the error was happening instead of here

Reply:
nbtItem.tag.name is intended to be empty. It's just somehow lost during the saving process, and I don't know prismarine-nbt enough to figure out what caused it to omit this empty string value.

Reply:
can you do if (nbtItem.tag && nbtItem.tag == undefined)

Reply:
did it